src/components/kpi/kpi.less

```diff
@@ -154,13 +154,30 @@
       text-align: center;
     }
 
-    .im-button.alert-caption {
+    .alert-caption {
       text-align: center;
       font-size: 11px;
-      width: 81%;
+      background: lighten(@kpi-negative-alert-label-background-color, 20%);
```

Comment:
I don't think we should use lighten here, when express projects are briefed to use a certain colour hexcode, it will cause problems :) You can use lighten in the variables.less to help you define the impac-angular default.